src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -248,7 +248,7 @@ private List<SdkResolver> GetResolvers(IList<SdkResolverManifest> resolversManif
                             MSBuildEventSource.Log.SdkResolverServiceLoadResolversStart();
                             newResolvers = _sdkResolverLoader.LoadResolversFromManifest(resolverManifest, loggingContext, sdkReferenceLocation);
                             _manifestToResolvers[resolverManifest] = newResolvers;
-                            MSBuildEventSource.Log.SdkResolverServiceLoadResolversStop(resolverManifest.DisplayName, newResolvers.Count);
+                            MSBuildEventSource.Log.SdkResolverServiceLoadResolversStop(resolverManifest.DisplayName ?? string.Empty, newResolvers.Count);
```

Comment:
I think this should never be null?

Reply:
For various of these: yeah, hopefully a future nullability pass will get rid of this stuff. But for now null checks are cheap so I put them in everywhere that flagged.